.github/ci_includes/image_versions.yml

```diff
@@ -20,7 +20,6 @@ BASE_JEKYLL: "{!{$BASE_IMAGES_REGISTRY_PATH}!}jekyll/jekyll:3.8@sha256:9521c8aae
 BASE_NGINX_ALPINE: "{!{$BASE_IMAGES_REGISTRY_PATH}!}nginx:1.15.12-alpine@sha256:57a226fb6ab6823027c0704a9346a890ffb0cacde06bc19bbc234c8720673555"
 BASE_NODE_16_ALPINE: "{!{$BASE_IMAGES_REGISTRY_PATH}!}node:16.13.0-alpine3.14@sha256:5277c7d171e02ee76417bb290ef488aa80e4e64572119eec0cb9fffbcffb8f6a"
 BASE_PYTHON_ALPINE: "{!{$BASE_IMAGES_REGISTRY_PATH}!}python:3.7.9-alpine3.12@sha256:8c224566c41fcc1674f9b78ff2a364a1209370a5b7e7619e8ead2bdcd0fd8b52"
-BASE_SCRATCH: "{!{$BASE_IMAGES_REGISTRY_PATH}!}spotify/scratch@sha256:db4cabf15c8b9eb70dabe1da385b6d9b2cac6d658b813fbb57dc5231ddd52420"
```

Comment:
I didn't get it 😅  Why have you deleted this?

Reply:
It's a generated file. The source is `.gitlab/image_versions.yml`

Seems a combination of two commits:
https://github.com/deckhouse/deckhouse/pull/715/files#diff-a3f8cd5c7599669a6a85935e737c3f488e7ca75da21c10283586888e7c37b0e1
https://github.com/deckhouse/deckhouse/pull/711/files

Reply:
Restored!